src/app/rpmostree-builtin-status.cxx

```diff
@@ -688,6 +688,27 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy, GVariant *child, gboolean
     }
   else
     g_print ("%s", checksum);
+
+  /* Get deployment index */
+  int index = 0;
+  GVariant *deployments = rpmostree_sysroot_get_deployments (sysroot_proxy);
+  GVariantIter iter;
+  g_variant_iter_init (&iter, deployments);
+  while (true)
+    {
+      g_autoptr (GVariant) other = g_variant_iter_next_value (&iter);
+      if (other == NULL)
+        {
+          index = -1;
+          break;
+        }
+
+      if (g_variant_equal (child, other))
+        break;
+      ++index;
+    }
+  g_assert (index >= 0);
+  g_print (" (Index: %d)", index);
```

Comment:
Minor: I think lowercase would be cleaner.

Reply:
Since all other keys in the output of `rpm-ostree status` are capitalized, I think writing `index: X` looks odd. But if you prefer that I can of course change it.